sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_operations/_patch.py

```diff
@@ -0,0 +1,294 @@
+    from azure.core.pipeline import PipelineResponse
+
+_SERIALIZER = Serializer()
+
+
+def build_schema_registry_get_schema_id_by_content_request( # pylint: disable=name-too-long
+    group_name: str,
+    name: str,
+    **kwargs: Any
+) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+    _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
+
+    content_type: str = kwargs.pop('content_type')
+    api_version: str = kwargs.pop('api_version', _params.pop('api-version', "2022-10"))
+    accept = _headers.pop('Accept', "application/json")
+
+    # Construct URL
+    _url = "/$schemaGroups/{groupName}/schemas/{name}:get-id"
+    path_format_arguments = {
+        "groupName": _SERIALIZER.url("group_name", group_name, 'str'),
+        "name": _SERIALIZER.url("name", name, 'str'),
+    }
+
+    _url: str = _url.format(**path_format_arguments)  # type: ignore
+
+    # Construct parameters
+    _params['api-version'] = _SERIALIZER.query("api_version", api_version, 'str')
+
+    # Construct headers
+    _headers['Content-Type'] = _SERIALIZER.header("content_type", content_type, 'str')
+    _headers['Accept'] = _SERIALIZER.header("accept", accept, 'str')
+
+    return HttpRequest(
+        method="POST",
+        url=_url,
+        params=_params,
+        headers=_headers,
+        **kwargs
+    )
+
+def build_schema_registry_register_schema_request(  # pylint: disable=name-too-long
+    group_name: str,
+    name: str,
+    **kwargs: Any
+) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+    _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
+
+    content_type: str = kwargs.pop('content_type')
+    api_version: str = kwargs.pop('api_version', _params.pop('api-version', "2022-10"))
+    accept = _headers.pop('Accept', "application/json")
+
+    # Construct URL
+    _url = "/$schemaGroups/{groupName}/schemas/{name}"
+    path_format_arguments = {
+        "groupName": _SERIALIZER.url("group_name", group_name, 'str'),
+        "name": _SERIALIZER.url("name", name, 'str'),
+    }
+
+    _url: str = _url.format(**path_format_arguments)  # type: ignore
+
+    # Construct parameters
+    _params['api-version'] = _SERIALIZER.query("api_version", api_version, 'str')
+
+    # Construct headers
+    _headers['Content-Type'] = _SERIALIZER.header("content_type", content_type, 'str')
+    _headers['Accept'] = _SERIALIZER.header("accept", accept, 'str')
+
+    return HttpRequest(
+        method="PUT",
+        url=_url,
+        params=_params,
+        headers=_headers,
+        **kwargs
+    )
+
+class SchemaRegistryClientOperationsMixin(GeneratedClientOperationsMixin):
+    """
+    Implement custom operations.
+    """
+
+    @distributed_trace
+    def get_schema_id_by_content(  # pylint: disable=inconsistent-return-statements
+        self,
+        group_name: str,
+        name: str,
+        schema_content: IO,
+        **kwargs: Any
+    ) -> None:
```

Comment:
I copied this operation from code generated from the swagger (so previously generated code), as per Izzy's recommendation. The two operations that were implemented here were not generated b/c they both support multiple content types in the request, and that's not something that's been added to the generation yet.

I'm not quite sure about the typing here. But I believe, after talking to Izzy, that this was None b/c it doesn't return anything by default. It only returns an object when we pass in a response hook/cls. I tried to find a way around this before, but I don't think there is one right now. If we wanted the return to be not None/Optional[...], it would have to return the same return type of the callback passed to `cls`.